xonsh/execer.py

```diff
@@ -185,15 +185,15 @@ def _parse_ctx_free(self, input, mode='exec'):
                     if prev_indent == curr_indent:
                         raise original_error
                 maxcol = self._find_next_break(line, last_error_col)
-                if line.startswith('$['):
-                    # if we have already wrapped this in subproc tokens
-                    # and it still doesn't work, adding more won't help
-                    # anything
-                    raise original_error
                 sbpline = subproc_toks(line,
                                        returnline=True,
                                        maxcol=maxcol,
                                        lexer=self.parser.lexer)
+                if sbpline.lstrip().startswith('$[$['):
```

Comment:
I think that line is a result of a merge from master (since it wasn't in the initial parent for this changeset).  I can rebase if you want that not to show up in the diff.